api/model/src/main/java/org/projectnessie/model/Content.java

```diff
@@ -34,7 +35,13 @@
 @Schema(
     type = SchemaType.OBJECT,
     title = "Content",
-    oneOf = {IcebergTable.class, DeltaLakeTable.class, IcebergView.class, Namespace.class},
+    anyOf = {
+      IcebergTable.class,
+      DeltaLakeTable.class,
+      IcebergView.class,
+      Namespace.class,
+      Map.class
```

Comment:
nit: this does not have a name in Swagger / Open API yaml:
```
    Content_V2:
      title: Content
      type: object
      properties:
        id:
          type: string
      anyOf:
      - $ref: '#/components/schemas/IcebergTable_V2'
      - $ref: '#/components/schemas/DeltaLakeTable_V2'
      - $ref: '#/components/schemas/IcebergView_V2'
      - $ref: '#/components/schemas/Namespace_V2'
      - type: object
```
not a big deal, but perhaps we can name it "Custom Content" (latger)

Reply:
Can't put a "name" for it :(